src/main/java/com/cinema/util/HibernateUtil.java

```diff
@@ -0,0 +1,23 @@
+package com.cinema.util;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+    private static SessionFactory sessionFactory = initSessionFactory();
+
+    public HibernateUtil() {
```

Comment:
see checklist: add a private default constructor to prevent HibernateUtil’s objects creating

Reply:
Done